### ISSUE TITLE
Feature: Add Edit Profile Button to Profile

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/app#2343]
### Type of feature

🍕 Feature

### Current behavior

Currently, users have to go into settings to change their profile. 

### Suggested solution

Let's add an Edit Profile button, similar to this, below the user profile and about the About section. This will impact #2342

<img width="378" alt="image" src="https://github.com/open-sauced/app/assets/34313413/e0654ae7-f807-4539-959b-9bfb47fe1c2a">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by isabensusan** [comment]
Added a fix to this on issue #2342  🫶🏻

**Comment by diivi** [comment]
I didn't notice this. Should I add an edit profile button in a follow-up PR?

**Comment by isabensusan** [comment]
@diivi yes please! that would be ideal 🙏

**Comment by BekahHW** [comment]
Hey @diivi I wanted to check in and see if you're still able to take this on.

**Comment by diivi** [comment]
Sorry, forgot about this.

@isabensusan 

![image](https://github.com/open-sauced/app/assets/41837037/18900f8b-56cc-4c44-8cb8-0e1472ca878f)


This is what the default variant button looks like for me. It doesn't have the borders as you shared in a separate thread. Should I add borders myself or leave it at the storybook version?

**Comment by devharsh2k4** [comment]
is it still open?

**Comment by nickytonline** [comment]
Yes it's still open. @devharsh2k4, please read the contributing guidelines.
